lazy-scripts/publish.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh -l
+
+set -euo pipefail
+
+if [ "$#" -ne 5 ]; then
+    echo "Usage: ./publish.sh [AWS_ACCOUNT_ID] [REPO_NAME] [IMAGE_NAME] [IMAGE_TAG] [DOCKERFILE_FULL_PATH]"
+    exit 1
+fi
+
+AWS_ACCOUNT_ID=${1}
+DOCKER_REPO_NAME=${2}
+IMAGE_NAME=${3}
+IMAGE_TAG=${4}
+DOCKER_REPO_HOST=${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com
```

Comment:
Get account ID from 
https://docs.aws.amazon.com/cli/latest/reference/sts/get-caller-identity.html

Reply:
Changed to use from sts caller indentity .